package.json

```diff
@@ -166,7 +166,7 @@
     "gulp-zopfli-green": "^3.0.1",
     "html-minifier": "^4.0.0",
     "husky": "^1.3.1",
-    "lint-staged": "^8.1.5",
+    "lint-staged": "^8.2.1",
```

Comment:
This is updated but I don't see it in `yarn.lock` or in the description of the PR, is this an intentional change?

Reply:
I was having problems with lint-staged preventing commit. It was not an intentional update. I ran npm install to make sure all dependencies were present.

Reply:
OK, please revert this in that case, or add a changelog and the updated `yarn.lock`. :-)

We use Yarn and not npm to manage our deps, so use `yarn install` instead of `npm install`

Reply:
Corrected! I'll make sure to use Yarn going forward.